### PR TITLE
Added missing Sphinx docstring

### PR DESCRIPTION
**Pull request by chrisdedman**

### CODE REVIEW

**Review by PenguinBoi12 — CHANGES_REQUESTED** [review]
It's great but the doc isn't the right format.
 https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html


To avoid repetition I'm not going to review the 2 other PRs about docstring since they all have the same issue.

Also, next time it could've been in one PR rather than multiple ones :)

**Comment by chrisdedman** [comment]
> It's great but the doc isn't the right format. We're using [this format](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html)
> 
> To avoid repetition I'm not going to review the 2 other PRs about docstring since they all have the same issue.
> 
> Also, next time it could've been in one PR rather than multiple ones :)

Outch! My bad, I will update that. 
yes, I wanted to do one PR, but for some reason, it create multiple one. Sorry about that

**Comment by PenguinBoi12** [comment]
Np it happens

File: bot/extensions/wikipedia_cog.py [hunk 1]
```diff
@@ -1,26 +1,40 @@
...
-from discord import ButtonStyle, ui
+from discord import ButtonStyle, ui, Embed
 from urllib.request import urlopen
 from urllib.parse import quote_plus
 from json import loads
-from discord import Embed
 
 
-def search_results(search):
+def search_results(search: str):
```
Comment (PenguinBoi12) [inline bot/extensions/wikipedia_cog.py:9]: You forgot the return type hint here.  I think it's a list of any (`-> List[Any]`)

**Review by PenguinBoi12 — CHANGES_REQUESTED** [review]
There's again the issue of consistency with typing. I know I'm annoying with that haha.

File: bot/extensions/wikipedia_cog.py (hunk 1 above)
Comment (chrisdedman) [reply bot/extensions/wikipedia_cog.py:9]: ahhh!!!!

**Review by PenguinBoi12 — APPROVED** [review]

**Pull request merged**